public/src/components/tests/variants/variantSummaryWebPreviewButton.tsx

```diff
@@ -26,7 +26,7 @@ const getChannelName = (testType: TestType, articleType: ArticleType): string =>
   } else if (testType === 'GUTTER' && articleType === 'Liveblog') {
     return 'gutter-liveblog';
   } else {
-    return testType.toLowerCase();
+    return testType.toLowerCase().replace('_', '-');
```

Comment:
the testType value is `LANDING_PAGE`, but for consistency here we make the query parameter `force-landing-page`